clouddriver-loadtest/src/main/resources/clouddriver-simulation.json

```diff
@@ -0,0 +1,22 @@
+{
```

Comment:
This can be overridden via
`-Dsimulation.config=/path/to/your/clouddriver-simulation.json` (doesn't have to be json)